### DISCUSSION TITLE
Generate A Summary for A Post in Hugo | /en/2017/08/hugo-post-summary/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1143] [Comments]
https://yihui.org/en/2017/08/hugo-post-summary/

[a comment by an automated account is not shown]

**Comment by haggishunk** [comment]
Thank you for teaching me to fish.  This was a very helpful and illuminating article that had me learn a bit more about the hugo ecosystem.

I found the theme I was using called out to a `{{ partial "post-summary.html" . }}` so I created a copy of the original `./layouts/partials/post-summary.html` from the theme into my hugo site base dir.  I tried to avoid replicating the html and templating from the theme in this override customization file and just using scratch set to change the `.Summary` variable, but in the end I had to remove the existing `{{ .Summary | safeHTML }}` and swap in the code snippet in your article. 

My site looks better already.

**Reply by yihui** [reply]
Good to know the post helped you a bit!
